test/libsolidity/Imports.cpp

```diff
@@ -320,6 +320,154 @@ BOOST_AUTO_TEST_CASE(shadowing_builtins_with_alias)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(inheritance_abi_encoder_mismatch_1)
+{
+	CompilerStack c;
+	c.addSource("A.sol", R"(
+	pragma solidity >=0.0;
+	pragma experimental ABIEncoderV2;
+
+	contract A
+	{
+		struct S { uint a; }
+		S public s;
+		function f(S memory _s) returns (S memory,S memory) { }
+	}
+)");
```

Comment:
Should be indented once (and everything above up to, but not including `c.addSource`).